howtos/release-procedure.md

```diff
@@ -21,20 +21,51 @@ The build process for [publish-prod](https://github.com/camunda/camunda-platform
 
 ## Perform a minor release
 
-Minor releases to Camunda Platform 8 happen twice a year in April and October.
+Minor releases to Camunda Platform 8 happen twice a year in April and October, and the documentation is versioned on the same cadence.
 
-To prepare for a minor release, you'll need to create a new verison.
+To prepare for a minor release, you'll need to create a new version.
 
 ### Create new version
 
-Technically, the current contents are frozen in `docs` and copied to `versioned_docs` with the corresponding version. The process can be triggered by this Docusaurus command:
+The versioning process copies a snapshot of the current documentation from the un-versioned source to a new versioned source location.
 
-```bash
-npm run docusaurus docs:version 8.1
+Because we are using [multiple docusaurus docs instances](./versioning.md#instances-docs-vs-optimize), we technically create _two_ new versions at time of product release: one for the main documentation, and one for the Optimize documentation.
 
-```
+Docusaurus creates the new versions in `versioned_docs` and `optimize_versioned_docs`. The contents in `docs` and `optimize` immediately become the documentation for the _next_ release.
 
-Create a PR with the changes and merge to `main`, confirming no build issues before moving to the release steps.
+To create the new versions:
+
+1. Make the `./hacks/cutNewVersions.sh` shell file executable, if it isn't already:
+
+   ```bash
+   > chmod +x ./hacks/cutNewVersions.sh
```

Comment:
A little CYA, but I'm 100% ok with not including Windows-based instructions here. I want it known before someone asks lol

Reply:
yeah....I went back and forth many times about whether this script should be bash or something cross-platform. I settled on bash for now because I don't see anyone but you or I running this next version release. By the time we do another, I might also need to make it node-based (in order to update the version mappings array via an abstract syntax tree) anyway.....in which case it'd be rewritten as cross-platform.

Blah blah blah words words words I agree, for now.